### ISSUE TITLE
General Furnace usage on forge 32.0.66 (1.16.1)

### ISSUE THREAD

**Issue by Blanketeer-X** [Qelifern/IronFurnaces#12]
When I was first getting a bunch of mods together, I figured that "Hey, why not this mod? I usually had fun with this mod.", and then I threw this in my modpack of choice. Got my adventure going on, and got my first iron furnace. So things are looking up for me, and I wanted to see how fast it smelted from furnace.

Long story short, by the time it finished smelting, it crashed with:
```
19:57:41.245
[m[1;31m[19:57:41] [Server thread/ERROR] [minecraft/MinecraftServer]: Encountered an unexpected exception
19:57:41.245
net.minecraft.crash.ReportedException: Ticking block entity
19:57:41.245
at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:859) ~[?:?] {re:classloading,pl:accesstransformer:B}
19:57:41.245
at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:791) ~[?:?] {re:classloading,pl:accesstransformer:B}
19:57:41.245
at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:81) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
19:57:41.245
at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:643) [?:?] {re:classloading,pl:accesstransformer:B}
19:57:41.245
at net.minecraft.server.MinecraftServer.lambda$func_240784_a_$0(MinecraftServer.java:230) [?:?] {re:classloading,pl:accesstransformer:B}

```

But hey, when I reloaded into it, the smelt was done. If you want me to do it with other furnaces if it's not a general thing, I can do that. Looking forward for the reply!

**Comment by XenoMustache** [comment]
Do you have a full crashlog you could give us? It would also be a great benefit if you would let us know if you experience this issue with other furnaces as well.

**Comment by Blanketeer-X** [comment]
[2020-07-17-1.log](https://github.com/Qelifern/IronFurnaces/files/4940965/2020-07-17-1.log)
I just copied the most important parts before, but here's the full one based on the log zip.

And I'll get on that in a hour or so

**Comment by Blanketeer-X** [comment]
[2020-07-18-2 (IronFurnaceTests).log](https://github.com/Qelifern/IronFurnaces/files/4941593/2020-07-18-2.IronFurnaceTests.log)
I didn't do all of them, but revisited the iron one and noticed something when I rejoined the crashed world. Apparently the world thinks I have the cobblestone unsmelted, but I got stone in  the smelted slot, so it still went on to continue smelting to create the crash that is this log.

_What I'm seeing here_ is that it's involved with the input/output slots only after it finishes the smelting attempt. Doesn't matter what kind of fuel or item we're using to smelt it, probably not for the furnace itself either. (And yes, I do have FastFurnaces, but it still crashed the same with or without it since the mod only affects the stone furnace)

And here's a silver one to be sure
[SilverFurnaceTest.log](https://github.com/Qelifern/IronFurnaces/files/4941598/SilverFurnaceTest.log)
